alpha/config/kConf.php

```diff
@@ -14,6 +14,7 @@
 set_include_path($include_path);
 
 require_once __DIR__ . '/../../infra/kEnvironment.php';
+require_once __DIR__ . '/../../infra/general/kString.class.php';
```

Comment:
remove the kString from here and add it to alpha/apps/kaltura/lib/request/infraRequestUtils.class.php